src/cascadia/TerminalSettingsModel/KeyMapping.h

```diff
@@ -69,6 +69,8 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         std::vector<Model::SettingsLoadWarnings> LayerJson(const Json::Value& json);
         Json::Value ToJson();
 
+        Windows::Foundation::Collections::IMap<Control::KeyChord, Model::ActionAndArgs> FetchGlobalHotkeys();
+

```

Comment:
Ugh, this'll conflict with #9621. I think it shouldn't be too difficult for me to resolve it though.